curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-076.md

```diff
@@ -8,9 +8,7 @@ isHidden: true
 ## Description
 <section id='description'>
 
-You change properties of a link when the mouse hovers them by using a <dfn>pseudo-selector</dfn> that looks like `a:hover { propertyName: propertyValue }`
-
-Change the color of the footer `Visit our website` link to be `brown` when a user hovers over it.
+Using the same style selector in the previous step, make the font size of the items and prices by using a value of `18px`.
```

Comment:
Using the same style selector in the previous step, make the font size of the items and prices larger by using a value of `18px`.